src/app/admintoolbar/admintoolbar.component.css

```diff
@@ -0,0 +1,9 @@
+.navbar{
+    margin-top: 1px;
+    padding: 1px;
+}
+.navbar-brand{
```

Comment:
Add space before start bracket in line 5.

Reply:
Done